frontend/src/features/contacts/contact/detail/Person.tsx

```diff
@@ -38,7 +38,7 @@ const PersonView: React.FunctionComponent<PersonViewProps> = ({ person }) => {
   const phoneTypes: Dictionary<string> = {};
   phoneTypes[ContactMethodTypes.WorkMobile] = 'Mobile';
   phoneTypes[ContactMethodTypes.WorkPhone] = 'Work';
-  phoneTypes[ContactMethodTypes.PersonalPhone] = 'Home';
+  phoneTypes[ContactMethodTypes.PersonalMobile] = 'Home';
```

Comment:
I believe this would still result in bugs. The database supports **TWO** kinds of personal phones:

* Personal Phone
* Personal Mobile

 We would need to support both

![image](https://user-images.githubusercontent.com/24322224/149244525-5adc2376-6796-4024-8806-02774221dfce.png)

Reply:
The current enums on the frontend support all available phone types. I think the read form should display a type for each possible phone type (even fax)

Reply:
the requirements specify the phone types to be displayed:
https://jira.th.gov.bc.ca/browse/PSP-2649?focusedCommentId=418270&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-418270

Reply:
I see in the code we are already using the label "Mobile" for work mobile phones so we would have to come up with a new string for personal mobile phones. Maybe this is something to ask the business?

Reply:
yep, please the link above.

Reply:
hmm I see - well it seems weird to me to be able to create a phone with type X and then not show in the details. I'd reconfirm with them. Showing N/A doesn't seem great in my opinion but I agree you are coding as per their requirements

Reply:
Ie, that the create screen allows the user to add an option that will not display on the view screen based on his response to Manuel's question.

Reply:
Yeah, that is a section that has been discussed multiple times since the confluence mapping was a tad confussing. That last link is the most straightforward definition of what the business needs.